pkg/rpm/Makefile

```diff
@@ -122,7 +122,8 @@ CONFIGURE_ARGS_COMMON=\
 
 CONFIGURE_ARGS=\
 	$(CONFIGURE_ARGS_COMMON) \
-	--njs
+	--njs \
+	--otel
```

Comment:
Does this need to change BUILD_DEPENDS_unit?
The same question stands for debian systems. 
It seems like "rustc" and "cargo" are missing from Build dependencies for packages

Reply:
it also seems like there is minimal cargo version is 1.78 which is not mentioned anywhere (https://github.com/rust-lang/cargo/issues/14655#issuecomment-2400237392)

Reply:
Though aren't we usually using the latest versions of cargo/rustc from rustup?

Reply:
wasm-wasi-component is also on version 4...

Reply:
Well, before this change I could simply do "cd pkg/deb && make unit". Now it fails mid-build, which is not the best way of failing

Reply:
Sadly we cant use rustc/cargo packages from the distros since they're outdated.  We have the same issue in the -wasm package.  That's why I added https://github.com/nginx/unit/commit/bf3d5759ebc17ef6c5997861bf2a73abb46af0a7 some time ago.